### PR TITLE
Task as/j305 pagination design according the design

### PR DESCRIPTION
**Pull request by qwiz1**
2021-09-01 21_47_40-Jabber

### CODE REVIEW

**Review by artem-totality — DISMISSED** [review]

**Review by JuliaDzyuba — DISMISSED** [review]

File: frontend/src/components/user-profile/user-profile.tsx [hunk 1]
```diff
@@ -209,14 +204,17 @@ const UserPage: React.FC = () => {
...
-            onSetPage={handleSetOffsetEpisodeFilter}
-            pageIndex={episodePagination.page}
-            pageSize={episodePagination.row}
-            episodesTotalCount={favoriteEpisodesCount}
-          />
+          ? <>
+            <FavouriteEpisodeTable
+              episodes={favoriteEpisodes}
+            />
+            <Pagination
```
Comment (what1s1ove) [inline frontend/src/components/user-profile/user-profile.tsx]: add pagination to table component

File: frontend/src/components/common/table/table.tsx [hunk 2]
```diff
@@ -2,28 +2,16 @@ import { useTable } from 'hooks/hooks';
 import { Column, useSortBy } from 'react-table';
 import styles from './styles.module.scss';
 import { getAllowedClasses } from 'helpers/helpers';
-import { Pagination } from './components/components';
+// import { Pagination } from 'components/common/common';
```
Comment (what1s1ove) [inline frontend/src/components/common/table/table.tsx]: add pagination component here

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Review by what1s1ove — CHANGES_REQUESTED** [review]
remove 'showing'
everything should be as in the design

File: frontend/src/components/common/table/table.tsx [hunk 3]
```diff
@@ -2,82 +2,82 @@ import { useTable } from 'hooks/hooks';
...
 type Props = {
   columns: Column[];
   data: unknown[];
-  onSetRow?: (row: number) => void;
-  onSetPage?: (page: number) => void;
-  pageIndex?: number;
-  pageSize?: number;
-  totalCountItems?: number;
-  hasPagination?: boolean;
+  pageCount: number;
```
Comment (what1s1ove) [inline frontend/src/components/common/table/table.tsx]: not all tables should be with pagination 
move it back

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**